.storybook/custom/prepareForInline.tsx

```diff
@@ -0,0 +1,31 @@
+import { StoryContext } from '@storybook/addons';
+import React from 'react';
+import { ContainerContext } from '../../docs/decorators/withContainer';
+import { decorators } from '../preview';
+
+export const prepareForInline = (
+  storyFn: () => JSX.Element,
+  ctx: StoryContext & ContainerContext
+) => {
+  const story = storyFn();
+  const content: unknown = ctx.getOriginal()(ctx.args);
+  const decorate = withDecorators(ctx);
+
+  if (typeof content === 'string') return decorate(<Html>{content}</Html>);
+
+  if (content instanceof Element) return decorate(<DomNode>{content}</DomNode>);
+
+  return story;
+};
+
+const DomNode = ({ children }: { children: Element }) => (
+  <Html>{children.innerHTML}</Html>
+);
+
+const Html = ({ children }: { children: string }) => (
```

Comment:
Probably minor, however this seems like you're rendering `<html>` but of course you _are not_.

Reply:
Ah, I see that now. Do you have a name suggestion? `<HtmlString>`? `<InnerHtml>`? `<RenderHtml>`?

Reply:
Could it be... `Container`? Or `InnerHtmlContainer`?

Reply:
Renamed to `HtmlContainer`, as `inner` is not necessarily accurate.